### PR TITLE
nic: add ipv6_disable capability

### PR DESCRIPTION
**Pull request by MarbolanGos**
Hello,

In the case you need to disable IPv6, the nic role is not yet capable to do this. I have written a small patch to add this feature.

### CODE REVIEW

File: roles/core/nic/templates/ifcfg.j2 [hunk 1]
```diff
@@ -89,3 +89,9 @@ MASTER={{ item.master }}
 {% if item.zone is defined and item.zone is not none %}
 ZONE={{ item.zone }}
 {% endif %}
+
+{# 9. Check if IPv6 has to be disabled #}
+{% if ipv6_disable is defined %}
```
Comment (btravouillon) [inline roles/core/nic/templates/ifcfg.j2:94]: This test will match `ipv6_disable: false` and disable ipv6 for this interface. I would use `{% if ipv6_disable | default(false) | bool %}` instead.
Moreover, I believe this should be `item.ipv6_disable`.

**Review by btravouillon — CHANGES_REQUESTED** [review]

File: roles/core/nic/templates/ifcfg.j2 (hunk 1 above)
Comment (oxedions) [reply roles/core/nic/templates/ifcfg.j2:94]: Dear @MarbolanGos

Any update on this PR ?

**Comment by MarbolanGos** [comment]
As this role (nic) is deprecated by nic_nmcli and that this feature is available by default in the next default role. This PR is not relevant anymore.

**Comment by oxedions** [comment]
Closing, as seen with @MarbolanGos

**Pull request closed without merging**